packages/electrode-react-webapp/package.json

```diff
@@ -5,12 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "echo \"Nothing to build. Just a placeholder\"",
-    "lint": "clap lint",
+    "lint": "xrun xarc/lint",
     "pre-test": "clap compile",
     "prepare": "clap compile",
-    "test": "clap test-only",
+    "test": "babel src-template -D -d template && mocha --extension ts,js,tsx,jsx,cjs,mjs -c --require @babel/register test/spec",
```

Comment:
FYI - 
Currently clap is not picking up configs from mocha.opts. Since I have to pass `--require @babel/register`, adding the command directly here. 
I would do further research on this in next PR so that we can use xrun